apps/smart-contracts/core/test/unit/PrePOMarket.test.ts

```diff
@@ -744,13 +768,37 @@ describe('=> prePOMarket', () => {
       const shortToRedeem = amountMinted
       const totalOwed = await calculateTotalOwed(longToRedeem, shortToRedeem, false)
 
-      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem)
+      await prePOMarket.connect(user).redeem(longToRedeem, shortToRedeem, user.address)
 
       expect(await longToken.balanceOf(user.address)).to.eq(amountMinted.sub(longToRedeem))
       expect(await shortToken.balanceOf(user.address)).to.eq(amountMinted.sub(shortToRedeem))
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed)
     })
 
+    it('correctly settles redemption with recipient != redeemer after market end', async () => {
```

Comment:
nit: (keep variables the same way around as the other test case)
```suggestion
    it('redeems if funder != recipient and market ended', async () => {
```